src/app/components/search/search-results/search-results.component.scss

```diff
@@ -73,4 +73,8 @@
       }
     }
   }
+
+  .adf-search-date-range .mat-form-field:not(:first-child) {
+    margin-top: 20px;
```

Comment:
should this be fixed in ADF too?

Reply:
No, the problem does not happen on demo-shell - it is only on ACA.

Reply:
Ok, thanks for confirming